.github/workflows/builds_and_tests.yml

```diff
@@ -134,6 +134,22 @@ jobs:
           name: libwasmvm
           path: ./api/libwasmvm.x86_64.so
 
+  build_static_lib:
+    needs: build_shared_library
+    runs-on: ubuntu-latest
+    steps:
+      - name: set up
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.17.4
+        id: go
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build docker image
+        run: cd builders && make docker-image-alpine
+      - name: Build static library for Alpine
+        run: make release-build-alpine
```

Comment:
Oh, this includes tests... How about leaving a comment about testing or updating the job name?

https://github.com/line/wasmvm/blob/main/Makefile#L74-L77

Reply:
How about leaving the job name only? (No need to write the comment.)
```
      - name: Build & Test static library
```